app.rb

```diff
@@ -0,0 +1,49 @@
+require 'sinatra/base'
+require './lib/chitter_model'
+
+class Peeps < Sinatra::Base
+
+  enable :sessions
+
```

Comment:
These routes are confusing and do not seem to be in the right order to gather the params you are expecting to be be gathering